lib/mint/http1.ex

```diff
@@ -834,7 +834,7 @@ defmodule Mint.HTTP1 do
       _ = Logger.debug(["Connection closed with data left in the buffer: ", inspect(conn.buffer)])
     end
 
-    :ok = conn.transport.close(conn.socket)
+    conn.transport.close(conn.socket)
```

Comment:
Can we be explicit and have

```suggestion
    _ = conn.transport.close(conn.socket)
```

?